landoscript/src/landoscript/actions/merge_day.py

```diff
@@ -85,9 +87,10 @@ async def run(github_client: GithubClient, public_artifact_dir: str, merge_info:
         # scenarios where we're uplifting one branch to another, and beginning a new
         # version number on the `to_branch`, which we declare with the `BASE` tag.
         if base_tag:
+            base_revision = await github_client.get_branch_head_oid(from_branch)
             base_tag_fmted = base_tag.format(major_version=bump_version.major_number)
             log.info(f"Adding base_tag: {base_tag_fmted}")
-            actions.extend(tag.run([base_tag_fmted]))
+            actions.extend(await tag.run(session, tag.GitTagInfo(revision=base_revision, tags=[base_tag_fmted])))
```

Comment:
Isn't there a race here if something lands between looking up the from_branch head and merging from_branch to to_branch?  Could we merge that base_revision to to_branch instead?

Reply:
Hm...yes, possibly. I'll need to check if shas are supported for the merge action (but I don't see why they wouldn't be...).

Reply:
(It works!)